src/AspNet.Security.OAuth.Naver/AspNet.Security.OAuth.Naver.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+  </PropertyGroup>
```

Comment:
```suggestion
  <PropertyGroup>
    <PackageValidationBaselineVersion>6.0.5</PackageValidationBaselineVersion>
    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
  </PropertyGroup>

  <!-- TODO Remove once published to NuGet.org -->
  <PropertyGroup>
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
  </PropertyGroup>
```

Reply:
I was just going to commit these to the PR myself and merge it, but I'm not being given permission to do so, so I'm guessing you didn't enable to option when you opened the PR.

This change is required for the CI to pass.